### PR TITLE
Add Android + ledger option

### PR DESCRIPTION
**Pull request by tomaskulich**

### CODE REVIEW

File: app/frontend/components/pages/login/hardwareAuth.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ const LoadByHardwareWalletSection = ({loadWallet}) => {
         alt: 'Ledger Nano S',
       }),
       h('div', {class: 'authentication-paragraph'}, 'Ledger Nano S'),
+      h('div', {class: 'authentication-paragraph small'}, 'also with Android device'),
```
Comment (j-matus) [inline app/frontend/components/pages/login/hardwareAuth.js]: This causes the Unlock with Trezor / Unlock with Ledger buttons to be no longer aligned, are we OK with that?

Comment (tomaskulich) [reply app/frontend/components/pages/login/hardwareAuth.js]: Probably not. Please think about it and come with some solution.

Comment (j-matus) [reply app/frontend/components/pages/login/hardwareAuth.js]: Adding a small authentication paragraph above the Unlock with Trezor button and setting its visibility to hidden is not very elegant but seems to work. Another option is to position the Trezor button by changing the `.button.trezor` style.

Comment (tomaskulich) [reply app/frontend/components/pages/login/hardwareAuth.js]: I fixed it somehow. It'll do IMO.

**Pull request merged**